### ISSUE TITLE
separating AccordianItem from Accordian doesn't work

### ISSUE THREAD

**Issue by iam-peekay** [daviferreira/react-sanfona#58]
When I try to put the AccordianItem component separate from the Accordian component, the AccordianItem doesn't expand. For example, my render function of the AccordianItem is defined as: 

```javascript
import { AccordionItem } from 'react-sanfona';

export default class Learnable extends Component {

....

render() {
return (
      <AccordionItem title={this.props.tags[0]} key={this.props.learnableid}  >
          <div>
            {this.props.text}
          </div>
      </AccordionItem>
    );
  }
}
```
and then my Accordian is defined as: 
```javascript
import Learnable from './Learnable';
import { Accordion } from 'react-sanfona';

export default class LearnableList2 extends Component {

... 

render() {
    const { learnables } = this.props;
    const learnablesList = learnables.map((learnable) => <Learnable key={learnable.id} learnableid={learnable.id} text={learnable.text} tags={learnable.tags} />);
    return (
      <Accordion allowMultiple >
        {learnablesList}
      </Accordion>
    );
  }
}
```

When I do this, only the title of each AccordianItem shows up, but if I try to expand the item, it doesn't expand. 

However, if I put the Accordian and AccordianItem in one file like you do in the demo examples, it works perfectly.

 Is there anything I'm doing wrong?

**Comment by 6ewis** [comment]
@iam-peekay I haven't used it ever but try to write this instead:   
```
const learnablesList = learnables.map(learnable => {
  return <Learnable key={learnable.id} learnableid={learnable.id} text={learnable.text} tags={learnable.tags} />;
})
```

**Comment by psimyn** [comment]
@iam-peekay you are missing `{...this.props} ` from the AccordionItem. Accordion adds an `onClick` prop to each item, so you need to pass those props as well as the ones you specify

Changed:
```
<AccordionItem title={this.props.tags[0]} key={this.props.learnableid}  {...this.props}>
```

**Comment by psimyn** [comment]
I pushed a probable fix to master - will get a 0.0.14 release out later today. Please re-open if still having the issue
